x/authz/keeper/keeper.go

```diff
@@ -129,7 +127,7 @@ func (k Keeper) DispatchActions(ctx context.Context, grantee sdk.AccAddress, msg
 				return nil, err
 			}
 
-			resp, err := authorization.Accept(sdkCtx, msg)
+			resp, err := authorization.Accept(ctx, msg)
```

Comment:
Now that we pass a context, we need to do like in the x/feegrant PR (break interface, or set environement in context)

Reply:
since we are using only time or height it might be safe. need to talk to @testinginprod

Reply:
ill open an issue on this, we will solve before release but lets unblock this since its blocked on a team call